utils.ps1

```diff
@@ -0,0 +1,298 @@
+# Need this variable as long as we support PS v2
+$ModuleBasePath = Split-Path $MyInvocation.MyCommand.Path -Parent
+
```

Comment:
All of the functions in this file need to be separated out into multiple PS1 files, we do not use a single file to hold mulitple functions anymore. The way these are being used would all be placed in the `dbatools\internal` directory.